event-adapter/glide.yaml

```diff
@@ -1,4 +1,4 @@
-package: k8s.io/k8s-stackdriver-adapter
+package: github.com/GoogleCloudPlatform/k8s-stackdriver/event-adapter
```

Comment:
If you add a Godep config in this commit, please delete glide.yaml and glide.lock.